README.md

```diff
@@ -0,0 +1,72 @@
+# ATD Bond Reporting
+
+ETL that is the backend of the ATD/PWD Bond Spending Dashboard in Power BI. 
+
+***
+
+## microstrategy_to_s3.py
+
+This script runs and then downloads a report from Microstrategy and then uploads it as a csv file in a S3 bucket. 
+
+### Config
+
+The `REPORTS` dict (`report_name`:`report_id`) at the top of this file stores what reports can be extracted. 
+
+### Power BI
+
+The purpose of this script is to make it easier to access Microstrategy reports in external applications like Power BI. The URL for all objects in the S3 bucket is default to publicly accessible. bond_data.py and bond_calculations.py build off of this. 
+
+### Running
+
+Run this script with the `-r` argument to provide the key of `REPORTS` dict you want to run. This will create a file in the S3 bucket with the same name.
+
+```
+$ python microstrategy_to_s3.py -r "name of your report"
```

Comment:
how does "name of your report" come into existence? you have to create it in Microstrategy? Would be good to explain that detail.

Reply:
This is indeed confusing, microstrategy needs report IDs so I have them stored as a dict `REPORTS` at the top of the microstrategy_to_s3.py file. The key of this dict is the "report name" and value is the report id. This name ends up getting passed along to the file name in S3, that way we can't have duplicate file names that get overwritten, too. I'll add this detail.

Reply:
👍 thanks